src/stores/game.js

```diff
@@ -364,8 +364,19 @@ export const useGameStore = defineStore('game', () => {
   }
   async function processThrees(chosenCard, game) {
     phase.value = GamePhase.MAIN;
-    lastEventCardChosen.value = chosenCard;
+    const three = game.resolved;
+    // Add three to scrap
+    scrap.value.push(three);
     await sleep(1000);
+
+    // Put selected card on top of scrap
+    lastEventCardChosen.value = chosenCard;
```

Comment:
The current implementation doesn't need or use this ref anymore, but I'd like to see if I can change the transition on the three target when it is moved to the top of the scrap pile. That would be a reason to keep this and use it to determine which card gets the special transition. 

Todo: either get the special transition for the three target working or remove the refs for maintaining three state